just added lab3.c to git

IgorBrat opened the pull request.

IgorBrat committed:
just added lab3.c to git

IgorBrat committed:
Update lab3.c

Deleted unnecessary library and changed rows 17, 21 (to make the project clearer for users)

IgorBrat committed:
Update lab3.c

Changed temporary variable name in rows 6-11 from "a" to "temp"

RoksolanaMazurak commented on lab3.c:
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <string.h>
+
+void reverseString(char s[])
```
please add comment to the function

IgorBrat replied:
fixed

IgorBrat committed:
Update lab3.c

zveres-ss commented on lab3.c:
```diff
@@ -0,0 +1,22 @@
+#include <stdio.h>
+#include <string.h>
+
+void reverseString(char s[])     // function which reverses inputed string
```
```suggestion
void reverseString(char input_string[])     // function which reverses inputed string
```

IgorBrat replied:
Done

zveres-ss commented on lab3.c:
```diff
@@ -0,0 +1,22 @@
+void main(void)
+{
+	printf("Input string: ");
+	char str[100];
```
make 100 as constant

IgorBrat replied:
Done

IgorBrat committed:
Update lab3.c

Changed name of vatiable in 4 line: "s" to "input_string"

Co-authored-by: Zenoviy Veres <viruslviv@gmail.com>

IgorBrat committed:
Update lab3.c

Changed name variable "s" to "input_string" in lines 5-13
Changed string length on line 19 to const variable LEN=100 (defined in line 3)

peopleAlreadyKnowWhoIAm commented on lab3.c:
```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
```
Make this program hack pentagon
